_posts/en/newsletters/2022-08-10-newsletter.md

```diff
@@ -0,0 +1,184 @@
+be sped up by caching the (successful) script and signature verifications
+that were done earlier. These caches must be part of the
+consensus engine, because consensus code refers to them.
+If the caching code has a bug, consensus may be
+lost if different nodes have diffrent cache sizes.
+Consensus code doesn't actually live in `src/kernel` yet; most of it
+is in `validation.cpp`.
+We refer to _consensus_ as consensus rules themselves, e.g. signature verification.
+And we're referring to signature caching as _consensus-critical_ functionality because,
+if we have an invalid signature cached, our node is no longer enforcing consensus rules."
+  a1link="https://bitcoincore.reviews/25527#l-45"
+
+  q2="What tools do you use for “code archeology” to understand the background of why
+a value exists?"
+  a2="`git blame filename`; `git log -p filename`; enter
+`commit:<commit-hash>` into the
+[pulls](https://github.com/bitcoin/bitcoin/pulls) page;
+use GitHub `Blame` button when viewing a file.
+You can also search for project symbols in the GitHub search bar."
+  a2link="https://bitcoincore.reviews/25527#l-132"
+
+  q3="This PR changes the type of `signature_cache_bytes` and
+`script_execution_cache_bytes` from `int64_t` to `size_t`.
+What is the difference between `int64_t`, `uint64_t`, and `size_t`,
+and why should a `size_t` hold these values?"
+  a3="The `int64_t` and `uint64_t` types are 64-bits (signed and unsigned,
+respectively) across all platforms and compilers. The `size_t` type
+is an unsigned integer that's
+guaranteed to be able to hold the length (in bytes) of any object
+in memory, including an array.
+The `size_t` type can be either 32 bits on a 32-bit memory system or
+64 bits on a 64-bit memory system. A `size_t` can also hold an array index,
+which is one of its most common uses.
+It also matches the size of a memory pointer."
+  a3link="https://bitcoincore.reviews/25527#l-163"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 0.12.0rc1][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25610][] opts-in the RPCs and `-walletrbf` to [RBF][topic rbf]
+  by default. This follows the update mentioned in
+  [Newsletter #208][news208 core RBF], enabling node operators to
```

Comment:
The first sentence throws me off, it's not clear to me how `-walletrbf` and the RPCs are changed. How about:

```suggestion
- [Bitcoin Core #25610][] sets by default the startup option `-walletrbf` and uses 
  by default the `replaceable` option for the RPCs `createrawtransaction` and `createpsbt`. 
  Transactions created via the GUI were already opt-in [RBF][topic rbf]
  by default. This follows the update mentioned in
  [Newsletter #208][news208 core RBF], enabling node operators to
```